project1-list/app/src/main/res/layout/secondaryviewlist_layout.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="vertical"
+                android:paddingTop="11dip"
+                android:paddingBottom="11dip"
+                android:paddingLeft="16dip"
+                android:paddingRight="16dip"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+                android:descendantFocusability="blocksDescendants">
+
+    <TextView android:id="@android:id/text1"
```

Comment:
I suggest making these id values more descriptive so it's very clear what they represent - something like `name_text` and `description_text`.